genie-client/src/main/java/com/netflix/genie/client/ClusterClient.java

```diff
@@ -129,21 +132,66 @@ public String createCluster(
      * @return A list of clusters.
      * @throws GenieClientException If the response received is not 2xx.
      * @throws IOException          For Network and other IO issues.
+     * @deprecated Use {@link #getClusters(String, List, List, Long, Long, Integer, SortAttribute, SortDirection,
+     * Integer)}
      */
     public List<Cluster> getClusters(
         final String name,
         final List<String> statusList,
         final List<String> tagList,
         final Long minUpdateTime,
         final Long maxUpdateTime
+    ) throws IOException, GenieClientException {
+        return this.getClusters(
+            name,
+            statusList,
+            tagList,
+            minUpdateTime,
+            maxUpdateTime,
+            null,
+            null,
+            null,
+            null
+        );
+    }
+
+    /**
+     * Method to get a list of all the clusters from Genie for the query parameters specified.
+     *
+     * @param name          The name of the cluster.
+     * @param statusList    The list of statuses.
+     * @param tagList       The list of tags.
+     * @param minUpdateTime Minimum Time after which cluster was updated.
+     * @param maxUpdateTime Maximum Time before which cluster was updated.
+     * @param pageSize      The maximum number of results returned
+     * @param sortAttribute The Application attribute used to sort
```

Comment:
Nit: 
```suggestion
     * @param sortAttribute The Cluster attribute used to sort
```
in this case? Same for command below as well.